freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/kerberosmgmt/model/ServicePrincipalRequest.java

```diff
@@ -0,0 +1,62 @@
+package com.sequenceiq.freeipa.api.v1.kerberosmgmt.model;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.sequenceiq.freeipa.api.v1.kerberosmgmt.doc.KeytabModelDescription;
+import com.sequenceiq.service.api.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel("ServicePrincipalV1Request")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class ServicePrincipalRequest {
+
+    @ApiModelProperty(KeytabModelDescription.ID)
+    private Long id;
```

Comment:
I don't get the purpose of this ID as it is a synchron call, the caller should receive the response immediately. If it would be async, then the service should return with an ID which could be queried by the caller

Reply:
I have added to track the request/response. Will remove it.